### PR TITLE
Bug fixes in Examples (PR #5 for issue #94)

### PR DESCRIPTION
**Pull request by anatoly-scherbakov**
# Why

YAML-LD samples have a few bugs.

# What

This PR is fixing those.

***
<a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/112.html" title="Last updated on Aug 16, 2023, 4:31 PM UTC (53f7bd4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/112/a82183a...53f7bd4.html" title="Last updated on Aug 16, 2023, 4:31 PM UTC (53f7bd4)">Diff</a>

### CODE REVIEW

**Comment by anatoly-scherbakov** [comment]
@gkellogg please check this out when you have a moment :slightly_smiling_face:

File: spec/index.html [hunk 1]
```diff
@@ -342,7 +342,7 @@ <h2>Introduction</h2>
 
     <div class="ednote">
       See YAML-LD description of this spec at
-      <a href="https://github.com/json-ld/yaml-ld/blob/main/spec/spec.yaml">
+      <a href="https://github.com/json-ld/yaml-ld/blob/main/spec/data/spec.yaml" target="_blank">
```
Comment (gkellogg) [inline spec/index.html:345]: It's cleaner to keep these references as relative to the spec, so when they're published, they show up properly. That would make it `<a href="data/spec.yaml">...</a>`.

**Review by gkellogg — CHANGES_REQUESTED** [review]
I suggested using relative links for yaml files. If there is a rendering issue, this might be addressed differently.

**Comment by anatoly-scherbakov** [comment]
Done so, — but it would seem it does not render. For instance: https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/data/spec.yaml does not work.

**Comment by gkellogg** [comment]
> Done so, — but it would seem it does not render. For instance: https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/data/spec.yaml does not work.

PR Preview doesn't have access to associated files, but you can see them via [GitHack](https://raw.githack.com/json-ld/yaml-ld/issue-94-convenience-contexts-examples/spec/index.html). This better reflects how the spec will render when merged.

**Review by gkellogg — APPROVED** [review]

**Pull request merged**